### ISSUE TITLE
'less' is not recognized as an internal or external command,

### ISSUE THREAD

**Issue by claws99** [dbcli/mycli#1088]
Hello I've installed this program using `pip install mycli`. All the commands are working fine, but none of the data (tables, databases, are being displayed on the screen).

Rather I keep getting the error:
`'less' is not recognized as an internal or external command,
operable program or batch file.`

![image](https://user-images.githubusercontent.com/83898279/190849468-5c2fdb0b-c4b6-499c-8dcf-4cf553b14b57.png)

**Comment by jbusk** [comment]
Yes, it seems that 'less' is required for the actual results to show. I suppose it'd be nice if there was a fallback option of just outputting the data without a pager, but in the meanwhile you can solve the problem by installing less (I installed it myself via chocolatey install less).

**Comment by MarvinXu** [comment]
you can use git bash shell instead

**Comment by Ravencentric** [comment]
You need to install [less](https://www.greenwoodsoftware.com/less/).
I used [scoop](https://scoop.sh/) to install it via `scoop install less` and that's pretty much all I had to do. Probably the easiest way to fix this. Although it should just be installed along with mycli imo.
